cli/src/build.rs

```diff
@@ -136,14 +139,19 @@ pub fn build_project(
                 .default(true)
                 .interact()?;
 
+            const WASM_TEMPLATE_DIR: Dir =
+                include_dir!("$CARGO_MANIFEST_DIR/src/template/mopro-wasm-lib");
+            let cwd = std::env::current_dir().unwrap();
+            copy_embedded_dir(&WASM_TEMPLATE_DIR, &cwd.join("mopro-wasm-lib"))?;
```

Comment:
actually it should not overwrite the `mopro-wasm-lib`
it should check if it exists, and not copy again if it exists

Reply:
we want users can write their customized functions to use bindgen in this folder
e.g.
```rs
#[wasm_bindgen]
pub fn hello() -> Result<JsValue, JsValue> {
    Ok(JsValue::from_str("Hello, world!"))
}
```
so it should not be overwritten